xiplot/plots/barplot.py

```diff
@@ -296,43 +266,13 @@ def render(
 
         return fig
 
-    @staticmethod
-    def create_layout(index, df, columns, config=dict()):
-        x_columns = [
-            c
-            for c in columns
-            if isinstance(df[c][0], Iterable)
-            or df[c].dtype in (int, np.int32, np.int64)
-        ]
-        y_columns = ["frequency"] + get_numeric_columns(df, columns)
-
-        jsonschema.validate(
```

Comment:
What happened to the config validation?

Reply:
Not validating lead to better error messages and partial backward compatibility. Now that everything is working I can re-add them (although I prefer "parse don't validate").

Reply:
I've re-added the validation